Cargo.toml

```diff
@@ -1,30 +1,32 @@
 [package]
-name    = "ulid"
+name    = "pgx_ulid"
```

Comment:
You shouldn't have a name conflict at all. Can you please describe the issue more?

Reply:
pgrx is trying to load the dynamic library ulid.so, but it is encountering the one from the ulid package and not from our extension, so I've renamed the crate to avoid any conflicts with the libraries generated